lib/MahoCLI/Commands/HealthCheck.php

```diff
@@ -23,11 +23,43 @@
 )]
 class HealthCheck extends BaseMahoCommand
 {
+    protected function checkComposer(OutputInterface $output): ?int
+    {
+        $result = Command::SUCCESS;
+        $loader = require MAHO_ROOT_DIR . '/vendor/autoload.php';
+
+        // if ($loader->getUseIncludePath() !== true) {
+        //     $result = Command::FAILURE;
+        //     $output->writeln('');
+        //     $output->writeln('<comment>Warning: Use include paths not enabled.</comment>');
+        //     $output->writeln('Run: composer config use-include-path true; composer dump;');
+        // }
+
```

Comment:
```suggestion
```

can we remove these?

Reply:
Absolutely, just left until I was 100% positive we don't need use-include-paths anymore. I'm 99% sure now, as composer will always lookup the classes with PSR-0 prefixes, or via the classmap.